lib/extensions/external_resources.ex

```diff
@@ -0,0 +1,62 @@
+defmodule BridgeEx.Extensions.ExternalResources do
+  @moduledoc """
+  Preload a set of resources marking them as "external" and provide the related getter functions.
+
+  ## Options
+
+    * `resources` (required): enumerable of resource names (atoms) and their paths (strings).
+      Each path is assumed to be relative to the module directory.
```

Comment:
Too much ? I know that by following good practices this should be enough but set a pretty big constraint on the use of the macro. Given the path is checked at compiled time  we could relax this and allow whatever path is given.

Reply:
This is a constraint that enforces keeping the files close to the module. When it is time, we could relax this constraint in a backwards-compatible way, but I don't see reasons to do it right now.